ProcessMaker/processmaker
# FOUR-8028 | Update Action Item Href Links in Ellipsis Menus

# Description
Ticket: [FOUR-8028](https://processmaker.atlassian.net/browse/FOUR-8028)

We need to update the href links for action items in the Ellipsis Menus that have already been integrated into the system.

Updated Ellipsis Menus:
	-Tasks/To Do Tasks
	-Designer/Processes
	-Designer/Processes(Archived)
	-Designer/Processes(Templates)
	-Designer/Scripts
	-Designer/Screens

# How to Test
1. Go to branch `feature/FOUR-8028` in core.
2. Go to each section of ProcessMaker listed in the description above.
	- View the Ellipsis Menu and test functionality (linking)
	- **For buttons that take the user to a different page, you should be able to right-click and open the link in a new tab**

# Related Tickets & Packages
- [FOUR-7955](https://processmaker.atlassian.net/browse/FOUR-7955) | Implement the New Ellipsis Menu for Specified Menus Across the Platform
- [FOUR-7878](https://processmaker.atlassian.net/browse/FOUR-7637) | Ellipsis Menu

# Code Review Checklist


[FOUR-8028]: https://processmaker.atlassian.net/browse/FOUR-8028?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[FOUR-7955]: https://processmaker.atlassian.net/browse/FOUR-7955?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[FOUR-7878]: https://processmaker.atlassian.net/browse/FOUR-7878?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
